portfolio/src/main/java/com/google/sps/servlets/Email.java

```diff
@@ -0,0 +1,19 @@
+package main.java.com.google.sps.servlets;
+
+/**
+ * Defines a class to describe an email and it's components
+ */
+
+ public class Email{
+//       Email new_email = new Email(id, title, timestamp);
```

Comment:
This comment may not add value, may be remove it?

Reply:
Yes I removed it